package.json

```diff
@@ -4,65 +4,10 @@
   "main": "dist/lib.js",
   "license": "MIT",
   "scripts": {
-    "clean": "rm -rf dist",
-    "build-development": "npm run clean && ./bin/build src/scout.js src/lib.js -v $npm_package_version",
-    "build-production": "npm run clean && ./bin/build src/scout.js src/lib.js -p -v $npm_package_version",
-    "build": "npm run verify && npm run build-development",
-    "format": "prettier --write \"**/*.js\"",
-    "lint": "eslint src test",
-    "verify": "npm run format && npm run lint",
-    "unit-test": "karma start --single-run",
-    "test": "npm run build && npm run unit-test"
+    "test": ""
```

Comment:
Nit: `test` needs to do _something_ or yarn doesn't like it:
`error An unexpected error occurred: "Script must exist".`